onnxruntime/test/unittest_main/test_main.cc

```diff
@@ -92,8 +92,8 @@ int TEST_MAIN(int argc, char** argv) {
   int status = 0;
 
   ORT_TRY {
-    ortenv_setup();
     ::testing::InitGoogleTest(&argc, argv);
+    ortenv_setup();
```

Comment:
This is very tricky. The ::testing::InitGoogleTest function also calls a lot of initialization functions in the tests, and the these functions may call into ONNX Runtime. Therefore ortenv should be initialized before that.